app/isomorphic/components/layouts/header/top-bar/index.js

```diff
@@ -6,7 +6,9 @@ import "./top-bar.m.css";
 const TopBar = () => (
   <div styleName="main-wrapper">
     <div className="container topbar-wrapper">
-      <AppLogo />
+      <h1>
```

Comment:
is it a good practise to keep the logo as h1, wouldn't this affect SEO?

Reply:
nope

Reply:
as in it won't affect seo

Reply:
@Harshith46 Maybe right. Please check that.